server_action_navigate/__manifest__.py

```diff
@@ -4,14 +4,15 @@
 
 {
     "name": "Server Actions - Navigate",
-    "version": "13.0.1.0.0",
-    "author": "GRAP, " "Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "GRAP, Ooops, Odoo Community Association (OCA)",
```

Comment:
One last thing. Bare migrations do not entitle for being recognized as "authors". Being in "contributors" is enough for now.

Reply:
done.